### ISSUE TITLE
Sprint commands often failing

### ISSUE THREAD

**Issue by purpleglade** [cwarwicker/discord-Writer-Bot#147]
I do not know if this is a discord-wide issue or something with the bot, but ever since the sprint command has switched to using /commands, we are the commands fail all the time. 

It's not uncommon for every sprint to have the sprint commands fail when adding their word count. Even launching sprints can have issues. 

This is the error that we get: 
![image](https://user-images.githubusercontent.com/6091343/155926583-46d22283-ffa6-42eb-af54-611225e96d4b.png)

I reinstalled the bot just in case, but it hasn't seemed to help. 

I am wondering if others are experiencing similar issues.

**Comment by cwarwicker** [comment]
Yes, there are lots of issues with the bot not being able to respond. I'm not sure why. Will try and look more into it once slash command migration is complete.

**Comment by livinginfictions** [comment]
Can confirm that this is a problem with the /wrote command as well.

**Comment by vossisboss** [comment]
Hello. We're seeing "The application did not respond" error as well as the occasional "Interaction error" on our server as well. I'd love to help track down bugs, but I'm having trouble getting the bot copy in the Writer-bot dev environment to behave properly in my test server. For example, the `?help` commands seem to be working but none of the slash commands are working. And it's a little hard to run tests without those working.

If anyone can confirm what permissions should be assigned to the bot that would be real helpful to know.

**Comment by livinginfictions** [comment]
> Hello. We're seeing "The application did not respond" error as well as the occasional "Interaction error" on our server as well. I'd love to help track down bugs, but I'm having trouble getting the bot copy in the Writer-bot dev environment to behave properly in my test server. For example, the `?help` commands seem to be working but none of the slash commands are working. And it's a little hard to run tests without those working.
> 
> If anyone can confirm what permissions should be assigned to the bot that would be real helpful to know.

Hey friend, i dont know much about the coding, but silly question in reference to yoh saying you cant get slash commands to work-have you given the bot the permission to use slash commands? It's a separate perm that you have to enable for them in their role.
Sorry if you've already done this, that was just my first thought.

**Comment by Starlight220** [comment]
The bot doesn't need any permissions given to it in server settings beyond what the invite link automatically gives it. To use slash commands, *users* (not the bot!) would need the `Use Application Commands` permission configured in server/channel/role settings.
The various `The application did not respond` errors are a result of the bot's long-existing performance issues, we're trying to fix those.

**Comment by livinginfictions** [comment]
Apologies if I'm being redundant but these "application did not respond" issues actually feel like they're getting worse? Maybe it's just because more commands have just been moved to slash stuff, but people in my server are having to try 3 or 4 times sometimes to get a command to go through/get processed. I'm getting at least one of these each time i try to use a project or wrote command. :/
